main.go

```diff
@@ -345,13 +345,12 @@ func main() {
 	log.Printf("Searching for Gemfile.lock with cocoapods gem")
 
 	// Check Gemfile.lock for CocoaPods version
-	gemfileLockPth := filepath.Join(podfileDir, "Gemfile.lock")
-	isGemfileLockExists, err := pathutil.IsPathExists(gemfileLockPth)
-	if err != nil {
+	gemfileLockPth, err := gems.GemFileLockPth(podfileDir)
+	if err != nil && err != gems.ErrGemLockNotFound {
```

Comment:
PLease update the error message "Failed to check Gemfile.lock at..." to not specifically refer Gemfile.lock, remove gemfileLockPth (as is empty string propably) or print the list of known gemfile names.

Reply:
Updated code.